R/rows-db.R

```diff
@@ -511,6 +534,13 @@ sql_returning_cols.tbl_dbi <- function(x, returning_cols, ...) {
   paste0("RETURNING ", returning_cols)
 }
 
+#' @export
+sql_returning_cols.tbl_duckdb_connection <- function(x, returning_cols, ...) {
+  if (!is_empty(returning_cols)) {
+    abort("DuckDB does not support the `returning` argument.")
+  }
```

Comment:
Emptiness is handled by the generic already?

```suggestion
  abort("DuckDB does not support the `returning` argument.")
```